SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/DummyObject.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#include "Interop/SpatialWorkerFlags.h"
+
+#include "DummyObject.generated.h"
+
+
+UCLASS()
+class UDummyObject : public UObject
```

Comment:
We should probably rename this object to be more descriptive about its association with the workflags testing